data/week1/4-reading-input.md

```diff
@@ -96,7 +97,8 @@ Once upon a time...
 </programming-exercise>
 
 ## Input String as a Part of Output
-Often we want to integrate user input within a larger piece of text as output. We refer to such a piece of text as a `String` and we already saw examples of strings when we printed output with the `System.out.println` command. When we want to join two pieces of text, or two `String`s together, we can use the addition (`+`) operator. The example below demonstrates a program that takes user input and prints it concatenated with another piece of text.
+To integrate user input with other text as output, we use the concept of `Strings`. A `String` is a piece of text for which we can use the `System.out.println` command to print. To join two pieces of text (`Strings`), we use the addition (`+`) operator. An example of this can be seen in the following program, that takes user input and combines it with another piece of text to print it as final output.
```

Comment:
", we use the concept of `Strings`." (eerste zin) is vreemd.

Suggestie:
To integrate user input with other text as output, we use the addition (`+`) operator to join two pieces of text (`Strings`) together. An example of this can be seen in the following program, that takes user input and combines it with another piece of text to print it as final output.